### ISSUE TITLE
Feature: Add Repo level context menu

### ISSUE THREAD

**Issue by shelbyKiraM** [desktop/desktop#9889]
### Describe the feature or problem you’d like to solve

- Right-click with repository list minimized shows same right-click menu as when expanded

### Proposed solution
- When right clicking, like here, it shows the right-click menu of when it's expanded and you find and right click on the repo,
 ```
Open in Command Prompt
Show in [Explorer]
Open in [Editor]
----
Remove Repository
```

- How will it benefit Desktop and its users?
    - Make it easier to jump to the current project quickly in another app.

### Additional context

- Screenshots of default behavior:
    - ![image](https://user-images.githubusercontent.com/202692/83203783-19500280-a0ff-11ea-96a4-3518fdd93a7a.png)
    - ![image](https://user-images.githubusercontent.com/202692/83205047-55d12d80-a102-11ea-8203-5ea1d28a9a97.png)

- My mock-up:
    - ![image](https://user-images.githubusercontent.com/202692/83204692-4f8e8180-a101-11ea-9f30-e9a52747facf.png)

**Comment by tierninho** [comment]
@marv3lls Thank you for the suggestion! I would probably use that shortcut myself, but I am not sure of the design or technical limitations regarding that area of the app. I will let this sit so others can weigh in with thoughts.

For reference:
Note, you access those some settings with one click via the `Repository` menu, or as you pointed out, by right-clicking on your repo from the repo list.

**Comment by niik** [comment]
This was fixed in https://github.com/desktop/desktop/pull/13664 :tada:

**Comment by shelbyKiraM** [comment]
> This was fixed in #13664 🎉

Let's GOOO-O-OOOO!!! 💖✌🏼‼️🐾
